stripe/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+pub struct CustomerID(pub String);
+
+pub fn create_customer(
+    conn: &mut ft_sdk::Connection,
+    session_id: &ft_sdk::auth::SessionID,
+    token: &str,
```

Comment:
This needs to be better named. Is this secret token, if so the name must include secret, if it is public token name must include that. Else we will not know if it is safe to log it etc.

Reply:
It is `access_token`. How do you get it? Is this a fixed value for a project, or dynamic per user etc?

Reply:
It is `stripe_secret_token`. This is a fixed value for a project. I'll rename this.

Reply:
done